geomstats/learning/exponential_barycenter.py

```diff
@@ -24,21 +24,21 @@ def _default_gradient_descent(
         Input points lying in the Lie Group.
     weights : array-like, shape=[n_samples,]
         default is 1 for each point
-        Weights of each point.
+        Weights associated to the points.
     max_iter : int, optional (defaults to 32)
-        The maximum number of iterations to perform in the gradient descent.
+        Maximum number of iterations to perform in the gradient descent.
     epsilon : float, optional (defaults to 1e-6)
-        The tolerance to reach convergence. The exstrinsic norm of the
+        Tolerance to reach convergence. The exstrinsic norm of the
         gradient is used as criterion.
     step : float, optional (defaults to 1.)
-        The learning rate in the gradient descent.
+        Learning rate in the gradient descent.
     verbose : bool
```

Comment:
Default: ...
Also missing default value for verbose

Reply:
Done.